metadata/configuration/concepts/1284_IPT_form.csv

```diff
@@ -0,0 +1,27 @@
+uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:fr,Short name:en,Data class,Data type,_version:1,_order:1284
+d369f5f3-70a8-43b1-9150-a55486c2cadb,,,Mois 1,Month 1,Mois 1,Month 1,Misc,N/A,,
+9bb92d03-f621-469a-883a-c2c87a0ea4c9,,,Mois 2,Month 2,Mois 2,Month 2,Misc,N/A,,
+b6624bc6-de83-4456-849e-899c490eb3f3,,,Mois 3,Month 3,Mois 3,Month 3,Misc,N/A,,
+10901d26-a115-4767-9821-9938e99787dd,,,Mois 4,Month 4,Mois 4,Month 4,Misc,N/A,,
+ceeaaae4-eb56-4134-9804-ae238935ec81,,,Mois 5,Month 5,Mois 5,Month 5,Misc,N/A,,
+b1969a83-f34b-4df3-b45a-7a7e2f6aac69,,,Mois 6,Month 6,Mois 6,Month 6,Misc,N/A,,
+d153e95d-e752-44dd-a887-cc249269408f,,,Sous Traitement,Under Treatment,Sous Traitement,Under Treatment,Misc,N/A,,
+ad0367b2-077a-4f8d-8f26-ff93c610ee66,,,Transféré In ,Transferred In,Transféré In,Transferred In,Misc,N/A,,
+aa474f33-5570-4d19-8f21-8a94de4815aa,,,Début du traitement TB,Start of TB Treatment,Début du traitement TB,Start of TB Treatment,Misc,N/A,,
+c732c133-f114-46b9-8d6b-b348737cfeed,,,Arrêt du Traitement INH,Stop INH Treatment,Arrêt du Traitement INH,Stop INH Treatment,Misc,N/A,,
+6ac5618b-20de-4a3c-a754-9dbe4d996be7,,,"6H, 9H (INH)","6H, 9H (INH)","6H, 9H (INH)","6H, 9H (INH)",Misc,N/A,,
+fe7c76cb-851f-4152-a1db-864667c5aca7,,,3HR (Daily),3HR (Daily),3HR (Daily),3HR (Daily),Misc,N/A,, 
+4d452283-bda5-4625-be08-49a0b2a4366d,,,4R,4R,4R,4R,Misc,N/A,,
+4f67eceb-7b91-4975-a6d9-ad4eee376a1a,,,1HP ,1HP ,1HP ,1HP ,Misc,N/A,,
+5a1678f6-716e-405e-9831-806163a6da9a,,,3 HP (Weekly),3 HP (Weekly),3 HP (Weekly),3 HP (Weekly),Misc,N/A,,
+c5f1b4e1-4ca3-4f8f-ad77-f72fc3660c4a,,,Preventive treatment of MDR-TB,Preventive treatment of MDR-TB,Preventive treatment of MDR-TB,Preventive treatment of MDR-TB,Misc,N/A,,
+c4e7476f-2386-486a-ab4d-0fa8820d3110,,"6H, 9H (INH);3HR (Daily);4R;3 HP (Weekly);1HP;Preventive treatment of MDR-TB",Protocole TPI,IPT Protocol,Protocole TPI,IPT Protocol,Misc,Coded,,
+44937ed1-cde6-4985-9811-2d7ee6d8d08f,,Month 1;Month 2;Month 3;Month 4;Month 5;Month 6,Mois de la Visite,Month of the Visit,Mois de la Visite,Month of the Visit,Misc,Coded,,
+f3040718-ea21-46c6-96fa-a120dd135f19,,,Date de la visite,Date of the visit,Date de la visite,Date of the visit,Misc,Date,,
```

Comment:
@mahaomolise92 can you please add the French translation of IPT Status which should be  Statut TPI in French